test-apps/display-performance-test-app/src/frontend/TestRunner.ts

```diff
@@ -988,9 +995,11 @@ function getRenderMode(vp: ScreenViewport): string {
 
 function getRenderOpts(curRenderOpts: RenderSystem.Options): string {
   let optString = "";
-  for (const [key, value] of Object.entries(curRenderOpts)) {
+  for (const propName of Object.keys(curRenderOpts)) {
+    const key = propName as NonFunctionPropertyNamesOf<RenderSystem.Options>;
```

Comment:
@DStradley compiler will now warn us if we delete or change the type of any ViewFlags property. It can't help us if we add a new property.